lib/semantics/check-do-concurrent.cc

```diff
@@ -420,29 +417,6 @@ class DoConcurrentContext {
         parser::Walk(
             std::get<parser::Block>(doConstruct.t), doConcurrentLabelEnforce);
         EnforceConcurrentLoopControl(*concurrent);
-      } else if (auto *loopBounds{
-                     std::get_if<parser::LoopBounds<parser::ScalarIntExpr>>(
-                         &optionalLoopControl->u)}) {
-        // C1120 - FIXME? may be checked before we get here
```

Comment:
What is the project's plan for auditing the code base for coverage of all Fortran standard constraints?

Reply:
Re: constraint checking coverage -- probably a conversation best had outside of a code review.

Reply:
None that I know of.